CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ## [1.16.1] - 2023-08-01
 
+### Added
+- recycling (#1638)
```

Comment:
This is again the released section 1.16.1. You need to add an unreleased section, see also https://github.com/OpenEnergyPlatform/ontology/wiki/How-to-release-a-new-ontology-version under no. 14

Reply:
Btw, that is something the release team as should have done directly after the release.